fedimint-client/src/lib.rs

```diff
@@ -1128,20 +1153,23 @@ impl ClientInner {
 
         let (tx, states) = partial_transaction.build(&self.secp_ctx, thread_rng());
 
-        Ok((tx, states, change_idx))
+        Ok((tx, states, change_range))
```

Comment:
**suggestion:** Please add a comment explaining the range, as it's even more non-obvious than it was before now. :D

Reply:
Done.